sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ReserveOfferRecommendation.java

```diff
@@ -16,11 +14,16 @@
     private final Operation operation;
 
     public ReserveOfferRecommendation(Offer offer, Resource resource) {
+        if (resource.hasDisk()) {
+            DiskInfo diskInfo = resource.getDisk().toBuilder()
```

Comment:
I have commented.  In the past the Resource passed in was appropriate to a RESERVE operation.  However now we have one way of generating a fully completed RESOURCE protobuf.  In the case of a disk this will include volume information relevant only to the CREATE operation.  So the volume information needs to be removed for the RESERVE operation.